safe/gui/tools/print_report_dialog.py

```diff
@@ -556,11 +563,18 @@ def template_chooser_clicked(self):
 
         .. versionadded: 4.3.0
         """
+        path = self.template_path.text()
+        if not path:
+            path = setting('lastCustomTemplate', '', expected_type=unicode)
```

Comment:
I would use `basestring`: https://github.com/inasafe/inasafe/commit/39eecfe1926e5d6014cdbf286dd45c96f89706c2
If you have `str`, it will fail with an exception because `str` ≠ `unicode`

I would very happy to move to Python 3. We have some accents in path, it's always scary for me.

Reply:
Ah sorry, I think @Gustry  is right.